csrc/index_compute.cpp

```diff
@@ -753,9 +753,8 @@ class UpdateLeafIndices : public IterVisitor {
     // Nothing need to be done when mappings for the output axes
     // already exist.
     if (index_map_.find(outer_id) != index_map_.end()) {
-      NVF_ERROR(
```

Comment:
This condition is not guaranteed to hold, and it gets exposed with more AlmostExact mappings. The code itself is considered (almost) obsolete.